src/app/modules/filters/filters-popover/filters-popover.component.ts

```diff
@@ -1,38 +1,56 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 
 @Component({
   selector: 'ccf-filters-popover',
   templateUrl: './filters-popover.component.html',
   styleUrls: ['./filters-popover.component.scss']
 })
-export class FiltersPopoverComponent {
-  filtersVisible = false;
+export class FiltersPopoverComponent implements OnInit {
   filtersBoxVisible = false;
+  filterContainer:HTMLElement|null;
```

Comment:
Not angular way
```suggestion
  @ViewChild('filterContainer', { read: ElementRef }) filterContainer: ElementRef<HtmlElement>;
```

ElementRef has a member 'nativeElement' that is the HTMLElement